pulp_ansible/tests/functional/constants.py

```diff
@@ -83,4 +83,5 @@
   version: ">=1.0.0,<=2.0.0"
   source: https://galaxy-dev.ansible.com/
 - testing.k8s_demo_collection
+- brightcomputing.bcm
```

Comment:
Where is this used?

Reply:
Oh wait this is what is being sync'd to ensure the code is working, is that right?

Reply:
exactly! to assure semver comparison is working